specification/09-coveio_abi.adoc

```diff
@@ -170,6 +170,10 @@ enable the created mappings until the TVM accepts the TDI in its TCB.
 
 The possible error codes returned in `sbiret.error` are shown below.
 
+`base_addr` value must be 4KB page aligned.
+
+`num_pages` value is the number of 4KB pages.
```

Comment:
I think we should change the argument here and use a `region_len` that should also be 4KB-aligned.

Reply:
yes, updated.